src/components/Layout.jsx

```diff
@@ -18,6 +18,7 @@ const Layout = ({ children }) => {
     <>
       <Reset />
       <GlobalStyles />
+      <Nav />
```

Comment:
Let's move this `Nav` into the header for now and replace what is currently in the header with it.

Reply:
I'll move this into the header, just wanted to note that the background for the `StyledHeader` is set as cfa-red, but on the current website the background is white.